src/pages/index.js

```diff
@@ -263,6 +283,155 @@ const IndexPage = () => (
+                  <p className="inline font-bold">Price</p>
+                </div>
+                <dd className="text-right inline">Staring from £100</dd>
+              </dt>
+            </dl>
+          </div>
+        </div>
+        {/* Treatment Content Title */}
+        <h2 className="text-2xl pt-8 pb-8">
+          <span className="text-3xl md:text-4xl text-red-500 font-bold">
+            About Content Block
+          </span>{" "}
+          - I dont know
+        </h2>
+        {/* About Content */}
+        <div className="grid grid-cols-1 grid md:grid-cols-2 gap-6">
+          <div>
+            <h2 className="text-2xl leading-tight tracking-wide pb-0 md:text-3xl">
+              Dr. Yalda Jamali is a qualified medical doctor specialising in
+              aesthetic medicine. She runs a successful clinic and mentors at
+              the Harley Academy.
+            </h2>
+          </div>
+          <div>
+            <p className="pb-4">
+              She graduated from the University of Manchester with a Bachelor of
+              Medicine, Bachelor of Surgery (MBChB) and holds a Level 7
+              postgraduate qualification in facial injectables. Dr Yalda is
+              passionate about ensuring aesthetics is a carefully regulated
+              industry, conducted with extreme precision, diligence and care.{" "}
+            </p>
+            <p className="pb-4">
+              She has a key artistic eye for detail. Her clients regard her as a
+              personable, results-oriented aesthetics doctor. Dr Yalda goes
+              above and beyond to ensure that her clients are completely
+              satisfied by their results. In addition to running a successful
+              clinic in Nottingham, Dr Yalda is a mentor at the Harley Academy
+              in London for medical professionals wishing to enter the
+              aesthetics field. In her capacity as a mentor, Dr Yalda provides
+              her insights and expert observations into the aesthetics industry
+              to the next generation of medical aesthetic professionals.
+            </p>
+            <p className="pb-4">
+              In addition to running a successful clinic in Nottingham, Dr Yalda
+              is a mentor at the Harley Academy in London for medical
+              professionals wishing to enter the aesthetics field. In her
+              capacity as a mentor, Dr Yalda provides her insights and expert
+              observations into the aesthetics industry to the next generation
+              of medical aesthetic professionals.
+            </p>
+            <p className="pb-4">
+              Dr Yalda also demonstrates her clinical skills at the GILD Clinic
+              in London, focusing on the practical application of facial
+              injectables and skin rejuvenation.
+            </p>
+            <img className="pt-5 h-20" src={signature} />
```

Comment:
Can we have proper alt text on all images please for accessibility. There are blind people out there